build.js

```diff
@@ -263,22 +237,22 @@ StyleDictionary.extend({
         {
           destination: "all.css",
           format: "css/variables",
-          filter: "isAlias",
-        },
-        {
-          destination: "colors.css",
-          format: "css/variables",
-          filter: "isAliasColor",
-        },
-        {
-          destination: "typography.css",
-          format: "css/variables",
-          filter: "isAliasTypography",
+          filter: (token) => {
+            if (!token.filePath.includes("alias.")) {
```

Comment:
If this targets `all.css` shouldn't it also include the component tokens?

Reply:
good catch 👍  I added [a PR](https://github.com/Frontify/arcade-tokens/pull/75) and asked Jamie to review it